backend/src/types.ts

```diff
@@ -964,6 +965,19 @@ export enum TolerationEffect {
   NO_EXECUTE = 'NoExecute',
 }
 
+export type Identifier = {
+  displayName: string;
```

Comment:
In the UX we are calling it "Resource label" instead of displayName, can we change here now so we don't have confusion later.

https://www.figma.com/proto/UFhLtsPxoETwd39pnMsASV/Hardware-profiles?node-id=310-2843&node-type=frame&t=pfzT3oPZXNHoHy6t-0&scaling=min-zoom&content-scaling=fixed&page-id=45%3A1342

Reply:
@dpanshug The CRD is created based on certain format its mentioned in the docs: https://docs.google.com/document/d/1kB78jtaqlj_-MUBpLVpGOSUUVmbGPs_XuMmCno0ezAU/edit?tab=t.0#heading=h.6hyvjhkv66pr

Reply:
calling it displayName better aligns with what it was called for other resources like accelerator profiles